keycloak/tasks/main.yaml

```
@@ -0,0 +1,61 @@
+---
+- name: Install prerequisites for Keycloak
+  ansible.builtin.apt:
+    name:
+      - openjdk-17-jdk
+      - unzip
+    state: present
+
+- name: Download Keycloak
+  ansible.builtin.get_url:
+    url: "https://github.com/keycloak/keycloak/releases/download/{{ keycloak_version }}/keycloak-{{ keycloak_version }}.zip"
+    dest: "/tmp/keycloak-{{ keycloak_version }}.zip"
+    mode: '0755'
```

```suggestion
    mode: '0644'
```
A zip has no reasons to be executable.

---

done